File: C-Sharp/Desafios Aritméticos Intermediários em C#/1-Notas e Moedas/solucao.cs
/* Leia um valor de ponto flutuante com duas casas decimais. Este valor representa um valor monetário. A seguir, calcule o menor número de notas e moedas possíveis no qual o valor pode ser decomposto. As notas consideradas são de 100, 50, 20, 10, 5, 2. As moedas possíveis são de 1, 0.50, 0.25, 0.10, 0.05 e 0.01. A seguir mostre a relação de notas necessárias.

Entrada
O arquivo de entrada contém um valor de ponto flutuante N (0 ≤ N ≤ 1000000.00).

Saída
Imprima a quantidade mínima de notas e moedas necessárias para trocar o valor inicial, conforme exemplo fornecido.

Obs: Utilize ponto (.) para separar a parte decimal. */



using System; 

class minhaClasse {

    static void Main(string[] args) { 

            double valor;
            int inteiro, auxNotas, auxMoedas;

            
            valor = Convert.ToDouble(Console.ReadLine());

            inteiro = (int)valor;
            valor *= 100;
            auxMoedas = (int)valor;


            Console.WriteLine("NOTAS:");
            Console.WriteLine("{0} nota(s) de R$ 100.00", inteiro/100);
            auxNotas = (inteiro % 100);
            //complete o código

            Console.WriteLine("MOEDAS:");
            Console.WriteLine("{0} moeda(s) de R$ 1.00", auxNotas / 1);
            auxMoedas %= 100;
            //complete o codigo

    }

}

File: Javascript/Desafios Iniciais JavaScript/1-Tabuada/solucao.js
/* Leia 1 valor inteiro N (2 < N < 1000). 
A seguir, mostre a tabuada de N:      
1 x N = N      2 x N = 2N        ...       10 x N = 10N

Entrada
A entrada contém um valor inteiro N (2 < N < 1000).

Saída
Imprima a tabuada de N, conforme o exemplo fornecido. */


let input = require("fs").readFileSync("stdin", "utf8");

let N = parseInt(input);

console.log("1 x " + N + " = " + (N * 1));
console.log("2 x " + N + " = " + (N * 2));
console.log("3 x " + N + " = " + (N * 3));
console.log("4 x " + N + " = " + (N * 4));
console.log("5 x " + N + " = " + (N * 5));
console.log("6 x " + N + " = " + (N * 6));
console.log("7 x " + N + " = " + (N * 7));
console.log("8 x " + N + " = " + (N * 8));
console.log("9 x " + N + " = " + (N * 9));
console.log("10 x " + N + " = " + (N * 10));

